projects/plugins/jetpack/extensions/blocks/cookie-consent/cookie-consent.php

```diff
@@ -47,6 +47,8 @@ function register_block() {
 		return;
 	}
 
+	notify_batcache_that_content_changed();
```

Comment:
Shouldn't this only be run inside `load_assets`, maybe?

Reply:
Sounds good! Tested and it works.